### DISCUSSION TITLE
Form shown on hover preview

### DISCUSSION THREAD

**Discussion by ItsRoryG** [kwsch/PKHeX#4136] [Ideas]
Hi,

Would it be possible for the hover preview to show the Pokemon form? If I am hovering over Minior, would it be possible to show its form i.e. Minior-Red in the preview? Or is this not shown due to space limitation?

Cheers

**Comment by Lusamine** [comment]
This is Showdown set standard, not any choice by PKHeX. The hover sets match Showdown set formatting, which does not use the name for the first form.

For example, Showdown sets call Single-Strike Urshifu only "Urshifu", and Rapid-Strike Urshifu is "Urshifu-Rapid-Strike". Single-Strike is the first form so it has no form suffix on it.

Since Red is the first form of Minior, it simply shows up as "Minior." The other colors of Minior will show their color.
